src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/begrunnelser/NasjonalEllerFellesBegrunnelse.kt

```diff
@@ -760,6 +760,17 @@ enum class NasjonalEllerFellesBegrunnelse : IBegrunnelse {
         override val sanityApiNavn = "avslagMottattI7Maaneder0824"
         override val begrunnelseType = BegrunnelseType.AVSLAG
     },
+
+    AVSLAG_ENDRINGSPERIODE_ALLEREDE_UTBETALT_SOKER {
```

Comment:
Nitpick, men i selve enumsa så har vi mulighet for ÆØÅ
```suggestion
    AVSLAG_ENDRINGSPERIODE_ALLEREDE_UTBETALT_SØKER {
```